stdlib/http/src/test/resources/testng.xml

```diff
@@ -21,6 +21,10 @@ under the License.
 
 <suite name="ballerina-test-suite">
 
+    <!--<listeners>-->
```

Comment:
Shall we add a comment explaining the process when debugging?

Reply:
Added with 6f65404